Autopilot/AttitudeManager/Attitude_States.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+* Attitude State CPP file
+*/
+#include "Attitude_States.hpp"
+#include "IMU.hpp"
+#include "airspeed.hpp"
+#include "PID.h"
+#include "OutputMixing.hpp"
+#include "SendInstructionsToSafety.hpp"
+#include "SensorFusion.hpp"
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+enum attitudeManagerState {
+    pathManagerInputMode,
+    PIDloopMode,
+    sendToSafetyMode,
+    failedMode
+};
+
+attitudeManagerState currentState;
+/***********************************************************************************************************************
+ * Prototypes
+ **********************************************************************************************************************/
+static attitudeManagerState pathManagerInput(void);
+
+
+static attitudeManagerState PIDloop(void);
+
+
+static attitudeManagerState sendToSafety(void);
+
+
+static attitudeManagerState failed(void);
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+void initAttitudeState()
+{
+    //Initialize sensor classes via smart pointer 
+    //Initialize PID class
+    PIDController PID; 
```

Comment:
Variables declared here will go out of scope as soon as initAttitudeState exits. You probably want to have them be file scope instead (or private if your'e going to use a c++ class model).